ecosystem/nft-metadata-crawler/nft-metadata-crawler-utils/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "nft-metadata-crawler-utils"
```

Comment:
What's the point of splitting out into 2 packages?

Reply:
Originally had all GCS and pubsub logic in a separate package so they can be used by uri retriever, parser, and any other processors

Reply:
But then you're calling this crawler-utils. I'd say that if it's customize code to crawler, merge them together b/c this is a single service. If you want to make some generic utils, maybe rename this and move this out. Currently I don't see your code as generic enough for other use cases so better to just combine.